### PR TITLE
Update vep tests

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

Switches pytests for nf-tests, and makes summary html output optional.

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/ensemblvep/vep/main.nf [hunk 1]
```diff
@@ -31,6 +31,7 @@ process ENSEMBLVEP_VEP {
     def file_extension = args.contains("--vcf") ? 'vcf' : args.contains("--json")? 'json' : args.contains("--tab")? 'tab' : 'vcf'
     def compress_cmd = args.contains("--compress_output") ? '' : '--compress_output bgzip'
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def stats_cmd = args.contains("--no_stats") ? '' : "--stats_file ${prefix}.summary.html"
```
Comment (nvnieuwk) [inline modules/nf-core/ensemblvep/vep/main.nf:34]: Is this an officially supported parameter?

Comment (maxulysse) [reply modules/nf-core/ensemblvep/vep/main.nf:34]: I'd rather add the `"--stats_file ${prefix}.summary.html` to the ext.args

Comment (ramprasadn) [reply modules/nf-core/ensemblvep/vep/main.nf:34]: Sounds good! I will remove it 👍🏻

File: modules/nf-core/ensemblvep/vep/main.nf [hunk 2]
```diff
@@ -46,7 +46,6 @@ process ENSEMBLVEP_VEP {
         --cache_version $cache_version \\
         --dir_cache $dir_cache \\
         --fork $task.cpus \\
```
Comment (maxulysse) [inline modules/nf-core/ensemblvep/vep/main.nf:48]:
```suggestion
        --fork $task.cpus
```

**Review by maxulysse — COMMENTED** [review]
Just one last change and we're good to go

**Review by rannick — APPROVED** [review]

**Pull request merged**